SXR/SDK/sxrsdk/src/main/java/com/samsungxr/SXRResourceVolume.java

```diff
@@ -192,6 +192,11 @@ else if (fname.startsWith("/sdcard/"))
             defaultPath = FileNameUtils.getParentDirectory(defaultPath);
             volumeType = SXRResourceVolume.VolumeType.ANDROID_SDCARD;
         }
+        else if (fname.startsWith("/storage/emulated/"))
```

Comment:
The path potentially could be anything. Please use Environment.getExternalStorageDirectory() to determine.